packages/components/checkbox/src/Checkbox.test.tsx

```diff
@@ -195,115 +194,52 @@ describe('Checkbox', () => {
     expect(onCheckedChange).toHaveBeenCalledWith(['baseball', 'soccer'])
   })
 
-  it('should render group with a label', () => {
-    render(
-      <FormField name="sports">
-        <FormField.Label>Sports</FormField.Label>
-
-        <CheckboxGroup>
-          <Checkbox value="soccer">Soccer</Checkbox>
-          <Checkbox value="baseball">Baseball</Checkbox>
-        </CheckboxGroup>
-      </FormField>
-    )
-
-    expect(screen.getByRole('group', { name: 'Sports' })).toBeInTheDocument()
-  })
-
-  it('should render group with required property', () => {
-    render(
-      <FormField name="sports" isRequired>
-        <FormField.Label>Sports</FormField.Label>
-
-        <CheckboxGroup>
-          <Checkbox value="soccer">Soccer</Checkbox>
-          <Checkbox value="baseball">Baseball</Checkbox>
-        </CheckboxGroup>
-      </FormField>
-    )
-
-    const groupEl = screen.getByRole('group', {
-      name: 'Sports',
-    })
-
-    const checkboxEls = within(groupEl).getAllByRole('checkbox')
-
-    expect(checkboxEls[0]).toHaveAttribute('aria-required', 'true')
-    expect(checkboxEls[1]).toHaveAttribute('aria-required', 'true')
-  })
-
-  it('should render group with helper message', () => {
-    render(
-      <FormField name="sports">
-        <FormField.Label>Sports</FormField.Label>
+  describe('with FormField', () => {
+    it('should render with a label', () => {
+      render(
+        <FormField name="sports">
+          <FormField.Label>Sports</FormField.Label>
 
-        <CheckboxGroup>
-          <Checkbox value="soccer">Soccer</Checkbox>
-          <Checkbox value="baseball">Baseball</Checkbox>
-        </CheckboxGroup>
-
-        <FormField.HelperMessage>Choose which sports you like</FormField.HelperMessage>
-      </FormField>
-    )
-
-    expect(
-      screen.getByRole('group', {
-        name: 'Sports',
-        description: 'Choose which sports you like',
-      })
-    ).toBeInTheDocument()
-  })
-
-  it('should render group with validation error', () => {
-    render(
-      <FormField name="sports" isInvalid>
-        <FormField.Label>Sports</FormField.Label>
-
-        <CheckboxGroup>
-          <Checkbox value="soccer">Soccer</Checkbox>
-          <Checkbox value="baseball">Baseball</Checkbox>
-        </CheckboxGroup>
-
-        <FormField.ErrorMessage>Sports combination is invalid</FormField.ErrorMessage>
-      </FormField>
-    )
+          <CheckboxGroup>
+            <Checkbox value="soccer">Soccer</Checkbox>
+            <Checkbox value="baseball">Baseball</Checkbox>
+          </CheckboxGroup>
+        </FormField>
+      )
 
-    const groupEl = screen.getByRole('group', {
-      name: 'Sports',
-      description: 'Sports combination is invalid',
+      expect(screen.getByRole('group', { name: 'Sports' })).toBeInTheDocument()
     })
 
-    expect(groupEl).toBeInTheDocument()
+    it('should render aria-attributes following FormField implementation', () => {
```

Comment:
- The more our tests resemble the way our users will interact with our components the better.
- I do not see the need of coupling this test with `FormField` concept.
- IMO this tests is too generic